internal/attestation/aws/issuer.go

```diff
@@ -21,14 +21,14 @@ import (
 	tpmclient "github.com/google/go-tpm-tools/client"
 )
 
+// Issuer for AWS TPM attestation
 type Issuer struct {
 	oid.AWS
 	*vtpm.Issuer
 }
 
 func NewIssuer() *Issuer {
-	awsIMDS := imds.New(imds.Options{})
-	GetInstanceInfo := getInstanceInfo(awsIMDS)
+	GetInstanceInfo := getInstanceInfo(imds.New(imds.Options{}))
```

Comment:
```suggestion
	getInstanceInfo := getInstanceInfo(imds.New(imds.Options{}))
```
As already mentioned, I would prefer to not instantiate the function, but use it directly.
Regardless of final implementation, the variable should start with a lowercase character